habitat_baselines/rl/models/simple_cnn.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import torch
+import torch.nn as nn
+
+from habitat_baselines.rl.ppo.utils import Flatten
+
+
+class SimpleCNN(nn.Module):
```

Comment:
Doc string needed, it is SimpleCNN adopted for Habitat actually. Maybe worth to name it `SimpleCNNPerception`.